front/migrations/20240403_memberships_no_more_revoked.ts

```diff
@@ -0,0 +1,20 @@
+import { frontSequelize } from "@app/lib/resources/storage";
+
+async function main() {
+  // We no longer rely on the "revoked" role for memberships.
+  await frontSequelize.query(`
+    UPDATE memberships
```

Comment:
All revoked roles already have a backfilled end date ([metabase](https://metabase.dust.tt/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJzZWxlY3QgKiBmcm9tIG1lbWJlcnNoaXBzIHdoZXJlIHJvbGUgPSAncmV2b2tlZCcgYW5kIFwiZW5kQXRcIiBpcyBudWxsIiwidGVtcGxhdGUtdGFncyI6e319LCJkYXRhYmFzZSI6NH0sImRpc3BsYXkiOiJ0YWJsZSIsInBhcmFtZXRlcnMiOltdLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fX0=)).

Now, revoked status does not exist, and we don't know their previous role so those memberships will me moved to "member"